tools/make_coupler_mosaic/Makefile.am

```diff
@@ -23,7 +23,8 @@ if WITH_MPI
 endif
 
 AM_CFLAGS = -I$(top_srcdir)/tools/libfrencutils \
-            $(NETCDF_CFLAGS)
+            $(NETCDF_CFLAGS) \
+            -I$(top_srcdir)/tools/fregrid
```

Comment:
I think other tools (make_coupler mosaic and the others) should not depend on the fregrid tool.  What is this tool picking up from the fregrid directory that the AM flags now have ```-I$(top_srcdir)/tools/fregrid``` ?

Reply:
The code does not compile without that include in AM_CFLAGS.  I think it's because create_xgrid.h needs global.h which is in the fregrid directory.

Reply:
@mlee03 So perhaps it makes more sense to put global.h in the same place as constant.h  - in directory libfrencutils ?

Reply:
Moved global.h to libfrenctools.  Ready for final review :)